<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="390" started-at="2016-10-06T20:36:51Z" finished-at="2016-10-06T20:36:51Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="390" started-at="2016-10-06T20:36:51Z" finished-at="2016-10-06T20:36:51Z">
      <class name="ExtentReport.SeleniumReport.App">
        <test-method status="PASS" signature="startReport()[pri:0, instance:ExtentReport.SeleniumReport.App@70124154]" name="startReport" is-config="true" duration-ms="108" started-at="2016-10-06T16:36:51Z" finished-at="2016-10-06T16:36:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method status="FAIL" signature="demoReportFail()[pri:0, instance:ExtentReport.SeleniumReport.App@70124154]" name="demoReportFail" duration-ms="11" started-at="2016-10-06T16:36:51Z" finished-at="2016-10-06T16:36:51Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at ExtentReport.SeleniumReport.App.demoReportFail(App.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:128)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:745)
	at org.testng.TestRunner.run(TestRunner.java:599)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demoReportFail -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:ExtentReport.SeleniumReport.App@70124154]" name="getResult" is-config="true" duration-ms="4" started-at="2016-10-06T16:36:51Z" finished-at="2016-10-06T16:36:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=demoReportFail status=FAILURE method=App.demoReportFail()[pri:0, instance:ExtentReport.SeleniumReport.App@70124154] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="demoReportPass()[pri:0, instance:ExtentReport.SeleniumReport.App@70124154]" name="demoReportPass" duration-ms="3" started-at="2016-10-06T16:36:51Z" finished-at="2016-10-06T16:36:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demoReportPass -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:ExtentReport.SeleniumReport.App@70124154]" name="getResult" is-config="true" duration-ms="0" started-at="2016-10-06T16:36:51Z" finished-at="2016-10-06T16:36:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=demoReportPass status=SUCCESS method=App.demoReportPass()[pri:0, instance:ExtentReport.SeleniumReport.App@70124154] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:ExtentReport.SeleniumReport.App@70124154]" name="endReport" is-config="true" duration-ms="237" started-at="2016-10-06T16:36:51Z" finished-at="2016-10-06T16:36:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- ExtentReport.SeleniumReport.App -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
